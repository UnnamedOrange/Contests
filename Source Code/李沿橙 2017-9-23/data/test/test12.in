5
8
1 1 1 1 999 999 999 1000 
7
1 8 9 10 11 12 13 
5
1 50 25 70 130 
50
377 808 182 502 372 675 557 909 894 657 167 770 142 719 254 275 301 744 888 641 76 7 70 999 87 247 839 790 690 971 863 325 576 220 236 302 846 810 718 432 149 213 472 131 316 614 587 22 116 592 
8
3 5 4 7 9 8 8 7 
