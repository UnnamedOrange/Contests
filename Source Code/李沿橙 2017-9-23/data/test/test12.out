1 1 1 1 999 999 1000 999 
1 8 10 11 12 13 9 
1 25 50 130 70 
7 22 70 76 116 131 142 149 167 213 220 236 247 254 301 302 316 372 432 502 576 587 592 614 657 675 690 718 744 770 790 808 839 846 863 888 909 999 971 894 810 719 641 557 472 377 325 275 182 87 
3 4 7 7 8 9 8 5 
